### DISCUSSION TITLE
[@passport-google-oauth 2.0] GoogleCallbackParameters snake_case outdated

### DISCUSSION THREAD

**Discussion by GabriMagic** [DefinitelyTyped/DefinitelyTyped#58185] [Issues with a @types package]
### Discussed in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/58005

<div type='discussions-op-text'>

<sup>Originally posted by **GabriMagic** January  6, 2022</sup>
Hi, I was working with Google authentication when I figured `GoogleCallbackParameters` exists, but when I tried to use it, I found that some properties came in camelCase (all except `id_token`, then I for undefined if I type them:

![image](https://user-images.githubusercontent.com/6409499/148331710-b9f9ffdd-afda-4739-a319-e280c4e8cf72.png)

Is this some outdated bug? Or maybe library inconsistencies? It would be nice to know if someone has noticed the same

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e605162d378f52f5cb8fccb137442320c876bf4d/types/passport-google-oauth20/index.d.ts#L280-L287

I am using this versions:

```
"passport-google-oauth20": "^2.0.0"
"passport": "^0.5.2"
```

Thanks in advance.</div>

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi, @GabriMagic. I created PR #58088

**Reply by GabriMagic** [reply]
Hi, I saw it, I added also a comment on the issue as I didn't get exactly documentation:

https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/58005

Should I do something more to the PR? 

Thank you
